mezzanine/__init__.py

```diff
@@ -1,2 +1,2 @@
 
-__version__ = "4.3.1"
+__version__ = "4.3.2"
```

Comment:
this will make it really annoying to be able to contribute this PR upstream :/

Reply:
Had to do it to ensure `pip install` actually uses the newest one. Does it support a `4.3.1utf8mb4` syntax. I'll try it.